knowledge-base/how-to-validate-report-parameters-on-the-client.md

```diff
@@ -0,0 +1,145 @@
+---
+title: How to validate report parameters on the client-side
+description: How to validate report parameters on the client-side
+type: how-to
+page_title: Client-side report parameters validation in HTML5 Report Viewer 
+slug: how-to-validate-report-parameters-on-the-client
+position: 
+tags: viewer,report,reporting, report parameters, validation, client-side validation, report parameters validation
+ticketid:
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+	<tr>
+		<td>Report Viewer</td>
+		<td>Web Viewers</td>
+	</tr>
+</table>
+
+
+## Description
+
+Client-side validation on a report parameter based on the value of another report parameter
+
+## Solution
+
+The following guide shows how to validate a report parameter in [HTML5 ASP.NET MVC Report Viewer](../html5-report-viewer). The approach could be applied to any other Web Report Viewer.
+
+1. Start by creating custom parameter editors by defining them through the [ParameterEditors method](https://docs.telerik.com/reporting/m-telerik-reportviewer-mvc-ireportviewerbuilder-parametereditors) when creating the report viewer widget:
```

Comment:
use relative paths to the reporting docs, for example ../m-telerik-reportviewer-mvc-ireportviewerbuilder-parametereditors.

You may edit the rest of the articles' links too.